### ISSUE TITLE
RECAMERA

### ISSUE THREAD

**Issue by tcpipchip** [ultralytics/yolov5#13701]
### Search before asking

- [x] I checked the docs, issues, and discussions and could not find an answer.

### Question

Hi

Did Ultralytics already implemented the algoritm to converte the .PT to .CVIMODEL ?

Today, we use the SEED algoritm...

But, it should me better all on ULTRALYTICS.

Thank you!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @tcpipchip, thank you for your interest in YOLOv5 🚀! Please visit our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5/) to get started, where you can find quickstart guides for simple tasks like [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/) all the way to advanced concepts like [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution/).
If this is a 🐛 Bug Report, please provide a **minimum reproducible example** to help us debug it.
If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Thanks for the question about converting `.pt` to `.cvimodel` for RECAMERA 🙂 This is an automated response, and an Ultralytics engineer will also assist soon.

## Requirements
[**Python>=3.8.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). To get started:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

## Environments
YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
-**Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
-**Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
-**Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
-**Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## Status
<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>
If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 [training](https://github.com/ultralytics/yolov5/blob/master/train.py), [validation](https://github.com/ultralytics/yolov5/blob/master/val.py), [inference](https://github.com/ultralytics/yolov5/blob/master/detect.py), [export](https://github.com/ultralytics/yolov5/blob/master/export.py) and [benchmarks](https://github.com/ultralytics/yolov5/blob/master/benchmarks.py) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

To help the team answer your `.pt` ➜ `.cvimodel` conversion question faster, please share:
-What exact YOLOv5 `.pt` model you’re starting from (e.g., custom-trained vs pretrained) and how it was produced (training command and logs if available)
-Your current export/conversion steps (commands/tools), including the “SEED algorithm” workflow you mentioned
-Target hardware + software environment (OS, Python, PyTorch, CUDA if used)
-Any error logs or output you see during conversion
-If you believe something is broken, please include a **minimum reproducible example (MRE)** that we can run locally (small model/weights + exact commands + expected vs actual result) 🧩

**Comment by tcpipchip** [comment]
sorry, wrong topic :(

**Comment by pderrenger** [comment]
No worries—YOLOv5 doesn’t currently export directly to `.cvimodel`; the closest supported handoff is exporting to ONNX (then converting with your vendor/SEED toolchain), see the [YOLOv5 model export docs](https://docs.ultralytics.com/yolov5/tutorials/model_export/) for the full list of supported formats.

**Comment by tcpipchip** [comment]
All solved using yolo to recamera and tdl sdk.

**Comment by pderrenger** [comment]
Glad you got it working with reCamera + the TDL SDK—if you can drop a quick note on the exact conversion path you used (so others can reproduce), that’d be great, and you can close this issue; for reCamera context see the [YOLO Models on Seeed Studio reCamera guide](https://www.ultralytics.com/blog/integrating-ultralytics-yolo-models-on-seeed-studios-recamera).

**Comment by tcpipchip** [comment]
Paula,

The problem was not on your side, but on my TPU MLIR side...

I had to include the --output parameters on model_transform.py

I use the yolo 

_yolo export model=best.pt format=onnx imgsz=640,640

**Comment by pderrenger** [comment]
Thanks for sharing the root cause—adding `--output` to `model_transform.py` on the TPU-MLIR side plus exporting YOLOv5 to ONNX via `yolo export model=best.pt format=onnx imgsz=640,640` is a great datapoint; if you can paste the exact `model_transform.py` command (and your TPU-MLIR/SDK versions), that’ll help others reproduce, and you can go ahead and close this issue.
